tools/testsys/Test.toml.example

```diff
@@ -0,0 +1,78 @@
+# This is an example testing configuration for TestSys, the tool that
+# is used to validate Bottlerocket builds.
+
+# This section contains configuration details for all testing
+[test]
+# The repo from `Infra.toml` that should be used for testing. It may be
+# useful to create a repo in `Infra.toml` that contains the infrastructure
+# needed for testing
+repo = "default"
+# The registry containing alternate TestSys agent images
+testsys-images-registry = "public.ecr.aws/bottlerocket-test-system"
+
+# The uri for the EKS resource agent that should be used.
+# An individual agent's provided URI will be used even if `testsys-images-registry` is present.
+eks-resource-agent-image = "public.ecr.aws/bottlerocket-test-system/eks_resource_agent:v0.0.2"
+
+# The test configuration is broken up into different levels depending on the variant.
+# Each configuration's value will be selected in the following order:
+# Arguments from the command line
+# Environment variables from the command line
+# '{variant}' configuration in `Test.toml`
+# '{family}-{flavor}' configuration in `Test.toml`
+# '{family}' configuration in `Test.toml`
+# '{platform}-{flavor}' configuration in `Test.toml`
+# '{platform}' configuration in `Test.toml`
+# TestSys default values
+
+# Configuration for all AWS variants (platform level configuration).
+[aws]
+agent-role = "<aws-testing-role>"
+
+# Configuration for all nvidia AWS variants on x86_64 (platform-flavor level configuration)
+[aws-nvidia.x86_64]
+instance-type = "p3.2xlarge"
+
+# Configuration for all nvidia AWS variants on aarch64 (platform-flavor level configuration)
+[aws-nvidia.aarch64]
+instance-type = "g5g.2xlarge"
+
+# Configuration for all `aws-k8s` variants testing (family level configuration).
+[aws-k8s]
+# A single role can be assumed by agents to test all `aws-k8s` variants in a separate
+# testing account.
+agent-role = "arn:aws:iam::<account_id>:role/<role_name>"
+
+# To test sonobuoy certified conformance in other AWS partitions, sonobuoy e2e
+# images need to be available in a registry in the targeted region (cn-north-1, us-gov-west-1).
```

Comment:
nit: this is not technically true if pulling images from ECR Public in GovCloud, and if reliability is not a concern then perhaps not for China either.

It might be better to document it as a way to set a custom conformance image, without reference to specific AWS partitions.